frameworks/cassandra/tests/config.py

```diff
@@ -105,7 +112,8 @@ def run_backup_and_restore(
     write_data_job = get_write_data_job(node_address=job_node_address)
     verify_data_job = get_verify_data_job(node_address=job_node_address)
     delete_data_job = get_delete_data_job(node_address=job_node_address)
-    verify_deletion_job = get_verify_deletion_job(node_address=job_node_address)
+    verify_deletion_job = get_verify_deletion_job(
```

Comment:
Looks like your col width is set to 80 and creating a lot of noise in the PR. How about 100?